server/templates/base.html

```diff
@@ -73,6 +73,21 @@
     globalThis.isCustomDC = {{ config['CUSTOM']|int }};
     globalThis.STAT_VAR_HIERARCHY_CONFIG = {{ config['STAT_VAR_HIERARCHY_CONFIG'] | tojson }};
   </script>
+  <script>
+    // Post the iframe's URL to the parent
```

Comment:
What does this block of code do?

Reply:
This block is to pass the iframe site url back to the parent

Reply:
is there  a way to extract this out of base.html to a ts/tsx file? if not, can you add some comments giving context as to what the intent is of the code (e.g., this supports the /dev/diff tool so navigating between the two sides i synchronized)

Reply:
Done